### ISSUE TITLE
Running TIme error.

### ISSUE THREAD

**Issue by LemonQC** [NilsDengler/view-point-pushing#1]
I  run the code, and find that
gymnasium.error.NameNotFound: Environment `ViewPointEnvGUI` doesn't exist.

**Comment by NilsDengler** [comment]
Hi @LemonQC ,
as stated in the setup.py file, this repository requires gym, not gymnasium, to run. However, you can change this by checking all the places in the code where we use gym and replacing the import with `import gymnasium as gym`. This should solve the problem. To find the occurrences of gym, you can simply run `grep -nr "gym" ` in the root directory.

**Comment by LemonQC** [comment]
I replace all the "import gym" with "import gymnasium as gym". It still can not work.

**Comment by LemonQC** [comment]
This is my environment.
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/bd4c0afb-d9ce-4641-91c5-db46129e5ab3)
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/41157ebd-9499-476a-a226-3ff649b26e75)

**Comment by LemonQC** [comment]
This is the log.
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/bb1a5430-e8b5-4ac7-88bd-1afd64de2a7e)

**Comment by NilsDengler** [comment]
Have you also changed it here: https://github.com/NilsDengler/view-point-pushing/blob/fcdb0a0aa3d257882bcb22cf34f7b33e808ac27a/shelf_gym/__init__.py#L1C9-L1C9

try to uninstall gym and try again

**Comment by LemonQC** [comment]
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/60679c77-3164-4711-b6c2-afd7d48dde95)

**Comment by LemonQC** [comment]
wow, I am silly. I solve it. Thanks

**Comment by NilsDengler** [comment]
Perfect!
Importing like this is not how it works :)

**Comment by LemonQC** [comment]
I got a new problem.
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/d67d5ff7-22c5-4610-a852-3f68644283ad)
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/0dc0a037-c672-4d4a-be8d-12834db72ea7)

**Comment by NilsDengler** [comment]
it's clearly a cuda/pytorch issue.
Have you installed everything the right way? I always use the following:
`conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia`

**Comment by LemonQC** [comment]
I use pip install -e . and still meet this problem. I reinstall according to your suggestions but still can not work. Can you show me your pip list

**Comment by NilsDengler** [comment]
absl-py==1.4.0
addict==2.4.0
asttokens==2.2.1
attrdict==2.0.1
attrs==22.2.0
backcall==0.2.0
cachetools==5.3.0
certifi @ file:///croot/certifi_1671487769961/work/certifi
charset-normalizer==3.1.0
click==8.1.3
cloudpickle==2.2.1
cmake==3.26.0
comm==0.1.2
ConfigArgParse==1.5.3
contourpy==1.0.7
cycler==0.11.0
dash==2.9.0
dash-core-components==2.0.0
dash-html-components==2.0.0
dash-table==5.0.0
debugpy==1.6.6
decorator==5.1.1
executing==1.2.0
Farama-Notifications==0.0.4
fastjsonschema==2.16.3
filelock==3.10.0
Flask==2.2.3
fonttools==4.39.1
google-auth==2.16.2
google-auth-oauthlib==0.4.6
grpcio==1.51.3
gym==0.21.0
gymnasium==0.28.1
h5py==3.8.0
idna==3.4
imageio==2.26.0
importlib-metadata==4.13.0
importlib-resources==5.12.0
imutils==0.5.4
ipykernel==6.21.3
ipython==8.11.0
ipywidgets==8.0.4
itsdangerous==2.1.2
jax-jumpy==1.0.0
jedi==0.18.2
Jinja2==3.1.2
joblib==1.2.0
jsonschema==4.17.3
jupyter_client==8.0.3
jupyter_core==5.2.0
jupyterlab-widgets==3.0.5
kiwisolver==1.4.4
lazy_loader==0.1
lit==15.0.7
Markdown==3.4.1
MarkupSafe==2.1.2
matplotlib==3.7.1
matplotlib-inline==0.1.6
mpmath==1.3.0
nbformat==5.7.0
nest-asyncio==1.5.6
networkx==3.0
numpy==1.24.2
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-cupti-cu11==11.7.101
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
nvidia-cufft-cu11==10.9.0.58
nvidia-curand-cu11==10.2.10.91
nvidia-cusolver-cu11==11.4.0.1
nvidia-cusparse-cu11==11.7.4.91
nvidia-nccl-cu11==2.14.3
nvidia-nvtx-cu11==11.7.91
oauthlib==3.2.2
open3d==0.17.0
opencv-python==4.7.0.72
packaging==23.0
pandas==1.5.3
parso==0.8.3
pbr==5.11.1
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.4.0
pkgutil_resolve_name==1.3.10
platformdirs==3.1.1
plotly==5.13.1
prompt-toolkit==3.0.38
protobuf==4.22.1
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pybullet==3.2.5
Pygments==2.14.0
pyparsing==3.0.9
pyquaternion==0.9.9
pyrsistent==0.19.3
python-dateutil==2.8.2
pytz==2022.7.1
PyWavelets==1.4.1
PyYAML==6.0
pyzmq==25.0.1
requests==2.28.2
requests-oauthlib==1.3.1
rsa==4.9
sb3-contrib==1.7.0
scikit-image==0.20.0
scikit-learn==1.2.2
scipy==1.9.1
six==1.16.0
-e git+https://gitlab.ipb.uni-bonn.de/dengler/pushing_softhand@570a56bff7b8ff2bafeabe877e57e5797364b6aa#egg=softhand_gym
stable-baselines3==1.7.0
stack-data==0.6.2
sympy==1.11.1
tenacity==8.2.2
tensorboard==2.12.0
tensorboard-data-server==0.7.0
tensorboard-plugin-wit==1.8.1
testresources==2.0.1
threadpoolctl==3.1.0
tifffile==2023.3.15
torch==2.0.0
tornado==6.2
tqdm==4.65.0
traitlets==5.9.0
trimesh==3.20.2
triton==2.0.0
typing_extensions==4.5.0
urllib3==1.26.15
wcwidth==0.2.6
Werkzeug==2.2.3
widgetsnbextension==4.0.5
zipp==3.15.0

**Comment by NilsDengler** [comment]
This is my list, but I don't think that it is because of the minor torch change

**Comment by LemonQC** [comment]
I run it successfully. But the utilization of cuda is still 0%. Maybe there exist some improvments.

**Comment by LemonQC** [comment]
I run the second script, but found.
![image](https://github.com/NilsDengler/view-point-pushing/assets/30914380/8fabd080-be19-4a57-bec3-f5d71d772f85)

**Comment by FeiZhou02** [comment]
> wow, I am silly. I solve it. Thanks

I also encountered the same issue. How did you resolve it?
